contractcourt/htlc_outgoing_contest_resolver.go

```diff
@@ -76,36 +76,6 @@ func (h *htlcOutgoingContestResolver) Resolve() (ContractResolver, error) {
 	default:
 	}
 
-	// We'll check the current height, if the HTLC has already expired,
-	// then we'll morph immediately into a resolver that can sweep the
-	// HTLC.
-	//
-	// TODO(roasbeef): use grace period instead?
-	_, currentHeight, err := h.ChainIO.GetBestBlock()
```

Comment:
More a question for my own understanding than review. 
Is the effect of removing this code that we no longer get the current block and check the expiry? And instead go straight into the for/select because that will receive the next block anyway (as per commit message)?

Reply:
Yes, the block notifier immediately sends the current block (even before the next) without delay. So that is effectively the same as `GetBestBlock` call that is removed here.